docs/stories/docker/mailhog-service.md

```diff
@@ -0,0 +1,14 @@
+# Mailhog service
+For development environments, the service mailhog helps to test emails locally, it's declared in the [docker-compose](../../../docker-compose.yml) and exports two ports, the first one is 1025 that it's used for smtp server, and the 8025 is a web interface, so you can see all test emails.
+
+For Inter-stack containers communication, this service can be used with its name and port, for example:
+1. If mailhog sendmail is not installed in the php environments (magento's services):
+  ```php
+  wget https://github.com/mailhog/mhsendmail/releases/download/v0.2.0/mhsendmail_linux_amd64
```

Comment:
Hrm, I'm not familiar with Mailhog's install on linux, have a doc to review?

Reply:
https://github.com/mailhog/MailHog#debian--ubuntu